syntheticsclientv2/integration_test.go

```diff
@@ -191,366 +194,579 @@ func TestLiveGetDevicesV2(t *testing.T) {
+	err = GetApiCheckV2(checkId, c)
 	if err != nil {
-		fmt.Println(err)
-	} else {
-		fmt.Println(reqDetail)
-		JsonPrint(res)
+		t.Fatal(err)
 	}
 
+	err = DeleteApiCheckV2(checkId, c)
 	if err != nil {
 		t.Fatal(err)
 	}
 
 }
 
-func TestLiveDeleteApiCheckV2(t *testing.T) {
-
+func TestLiveCreateApiCheckV2(t *testing.T) {
 	//Create your client with the token
 	c := NewClient(token, realm)
 
-	// Make the request with your check settings and print result
-	res, err := c.DeleteApiCheckV2(3435)
+	checkId, err := CreateApiCheckV2(createApiV2Body, c)
 	if err != nil {
-		fmt.Println(err)
-	} else {
-		JsonPrint(res)
+		t.Fatal(err)
 	}
 
+	log.Printf("Api Check ID: %d", checkId)
+}
+
+func TestLiveGetApiCheckV2(t *testing.T) {
+
+	//Create your client with the token
+	c := NewClient(token, realm)
+
+	err := GetApiCheckV2(1111, c)
 	if err != nil {
 		t.Fatal(err)
 	}
 
 }
 
-func TestLiveCreatePortCheckV2(t *testing.T) {
+func TestLiveUpdateApiCheckV2(t *testing.T) {
+
+	//Create your client with the token
+	c := NewClient(token, realm)
 
-	err := json.Unmarshal([]byte(createPortCheckV2Body), &inputPortCheckV2Data)
+	err := UpdateApiCheckV2(1111, updateApiCheckV2Body, c)
 	if err != nil {
 		t.Fatal(err)
 	}
 
+}
+
+func TestLiveDeleteApiCheckV2(t *testing.T) {
+
 	//Create your client with the token
 	c := NewClient(token, realm)
 
-	fmt.Println(inputPortCheckV2Data)
+	err := DeleteApiCheckV2(1111, c)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+}
+
+func CreatePortCheckV2(test string, c *Client) (int, error) {
+
+	err := json.Unmarshal([]byte(test), &inputPortCheckV2Data)
+	if err != nil {
+		return 0, err
+	}
 
 	// Make the request with your check settings and print result
 	res, reqDetail, err := c.CreatePortCheckV2(&inputPortCheckV2Data)
 	if err != nil {
-		fmt.Println(err)
-	} else {
-		fmt.Println(reqDetail)
-		JsonPrint(res)
+		return 0, err
 	}
+	fmt.Println(reqDetail)
+	JsonPrint(res)
+
+	return res.Test.ID, nil
+}
 
+func UpdatePortCheckV2(checkId int, test string, c *Client) error {
+
+	err := json.Unmarshal([]byte(test), &updatePortCheckV2Data)
 	if err != nil {
-		t.Fatal(err)
+		return err
 	}
 
+	// Make the request with your check settings and print result
+	res, reqDetail, err := c.UpdatePortCheckV2(checkId, &updatePortCheckV2Data)
+	if err != nil {
+		return err
+	}
+	fmt.Println(reqDetail)
+	JsonPrint(res)
+	
+	return nil
 }
 
-func TestLiveGetPortCheckV2(t *testing.T) {
+func GetPortCheckV2(checkId int, c *Client) error {
+	// Make the request with your check settings and print result
+	res, _, err := c.GetPortCheckV2(checkId)
+	if err != nil {
+		return err
+	} 
+	JsonPrint(res)
+
+	return nil
+}
+
+func DeletePortCheckV2(checkId int, c *Client) error {
+	// Make the request with your check settings and print result
+	res, err := c.DeletePortCheckV2(checkId)
+	if err != nil {
+		return err
+	}
+	JsonPrint(res)
+	
+	return nil
+}
+
+func TestLivePortCheckCreateUpdateAndDeleteV2(t *testing.T) {
```

Comment:
End to end test of Port Check:
1. Create
2. Update
3. Get
4. Update with original payload
5. Get
6. Delete